config/routes.php

```diff
@@ -174,7 +174,7 @@ function (RouteCollector $route): void {
             function (RouteCollector $route): void {
                 // Shirts
                 $route->addGroup(
-                    '/shirt',
+                    '/goodie',
```

Comment:
und für die kleinen changes ein ganzer commit ???
grummel grummel

Reply:
Just to not confuse you...